redash/controllers.py

```diff
@@ -450,7 +450,7 @@ def delete(self, visualization_id):
 class QueryResultListAPI(BaseResource):
     @require_permission('execute_query')
     def post(self):
-        params = request.json
```

Comment:
Got it. I think there is a way to tell Flask to return json regardless of
the content type header. But keep it as is, I'll fix the test helper and
update it later.
On May 15, 2015 7:42 AM, "stanhu" <notifications@github.com> wrote:

> In redash/controllers.py
> <https://github.com/EverythingMe/redash/pull/432#discussion_r30384750>:
>
> > @@ -450,7 +450,7 @@ def delete(self, visualization_id):
> >  class QueryResultListAPI(BaseResource):
> >      @require_permission('execute_query')
> >      def post(self):
> > -        params = request.json
>
> I tried to keep it, but the test case seems to serialize it into a format
> that can't be parsed by request.json:
>
>
> https://github.com/EverythingMe/redash/blob/master/tests/test_controllers.py#L33
>
> Is it missing a header of some sort? I didn't have time to research. I
> just changed it to be consistent with the rest of the code in the
> controllers.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EverythingMe/redash/pull/432/files#r30384750>.
>